src/app/errors/inbox-linshare-errors.service.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .factory('inboxLinshareErrors', inboxLinshareErrors);
```

Comment:
This will cause an issue in Webpack production mode. It should be `inboxLinshareErrorsFactory`